modules/social_features/social_content_block/social_content_block.services.yml

```diff
@@ -8,6 +11,12 @@ services:
     class: Drupal\social_content_block\ContentBuilder
     arguments:
       - '@entity_type.manager'
-      - '@database'
       - '@module_handler'
       - '@string_translation'
+      - '@plugin.manager.content_block'
+      - '@database'
```

Comment:
Changing the argument of services is a breaking change because it breaks extending services. Adding an argument may be possible in a minor version if there's a fallback for when the service is not present. Changing the order is always a breaking change due to mismatched types.

Reply:
Fixed